common/app/common/utils/FormUtils.java

```diff
@@ -0,0 +1,39 @@
+package common.utils;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.models.AddressBuilder;
+import play.data.Form;
+
+import javax.annotation.Nullable;
+
+public final class FormUtils {
+
+    private FormUtils() {
+    }
+
+    public static Address extractAddress(@Nullable final Form<?> form, final String suffix) {
+        if (form != null) {
+            final CountryCode country = countryOrUndefined(form.field("country" + suffix).value());
```

Comment:
Yep, I can also enable this extractor in this utils class, so that it can be used elsewhere.